new/utils/newProjectInstructions.js

```diff
@@ -5,17 +5,20 @@ const fs = require('fs')
 const boxen = require("boxen");
 const store = require('../store')
 const name = process.argv[3];
+const chalk = require('chalk')
+
+const link = 'blixjs.com'
 
 const options = {
-  mongooseModel: { command: "model", example: "npm run model User email:String age:Number", use: "Creates Mongoose model" },
-  postgresModel: { command: "model", example: "npm run model User email:string age:number", use: "Creates Bookself model" },
-  reactComponent: { command: "component", example: "npm run component <name>", use: "Creates a stateful or stateless React component and CSS file in a folder within src/" },
-  reactRouterComponent: { command: "component", example: "npm run component <name>", use: "Creates a stateful or stateless React component and CSS file in a folder within src/components/"},
-  reduxComponent: { command: "component", example: "npm run component <name>", use: "Create a React component, Redux container, and CSS file in a folder within src/"},
-  view: { command: "view", example: "npm run view about", use: "Create a stateful or stateless React component in src/views/ and add the route to the src/App.js router file. Also imports components in the new view."},
-  reactRouterReduxComponent: { command: "component", example: "npm run component <name>", use: "Create a React component, Redux container, and CSS file in a folder within src/components"},
-  action: { command: "action", example: "npm run action", use: "Create new action, and creates or appends a Reducer, and adds the action to selected containers"},
-  controller: { command: "controller", example: "npm run controller <name>", use: "Quickly create api/v1 get|post|put|delete endpoints for a resource"},
+  mongooseModel: { command: "model", example: "blix generate model <ModelName> [fieldName]:[Type] [fileName]:[Type]", use: "Creates Mongoose model" },
```

Comment:
Updated examples to blix generate

Reply:
I like this for now, but as we continue to scale, we should consider removing these longer examples and put them in the man page / in the docs.